.travis.yml

```diff
@@ -140,6 +140,15 @@ jobs:
         - travis_retry ./scripts/if_changed.sh ./scripts/build.sh $PROJECT $PLATFORM
         - travis_retry ./scripts/if_changed.sh ./scripts/pod_lib_lint.sh GoogleDataTransport.podspec
 
+    # GoogleDataTransport integration tests using the default Xcode version.
```

Comment:
Why is the same block added here and below?

Reply:
AFAICT, I had to factor out the integration test run into its own travis 'project' so I could tell travis to ignore it if it failed. If I added it only into the allow_failures section, travis wouldn't run it at all. All the projects listed in that section are written up here, except for the Firestore fuzz test which runs on a cronjob.

Could I write the allow_failures rule like so?:

```
  - stage: test
      env:
        - PROJECT=GoogleDataTransportIntegrationTest PLATFORM=iOS METHOD=xcodebuild
```

Reply:
see below